src/lib/utils/onboardHelper.ts

```diff
@@ -7,7 +7,11 @@ import { NETWORKS, TOKEN_NAMES } from '~lib/utils/envHelper';
 
 export const initOnboard = () => {
   const injected = injectedModule();
-  const walletConnect = walletConnectModule({ dappUrl: 'https://app.ssv.network/', projectId: config.ONBOARD.PROJECT_ID, optionalChains: [NETWORKS.MAINNET, NETWORKS.GOERLI, NETWORKS.HOLESKY] });
+  const walletConnect = walletConnectModule({
+    dappUrl: 'https://app.ssv.network/',
```

Comment:
I propose to use here domain name from window location object. 
It might be the reason when some wallets could potentially have issues connecting to the app

Reply:
done